cert/cert.go

```diff
@@ -482,6 +481,33 @@ func (spec *Spec) IsChangedOnDisk(path string) bool {
 	return specStat.ModTime().After(st.ModTime())
 }
 
+func (spec *Spec) CertExpireTime() time.Time {
```

Comment:
Same as above, needs a doc comment above in the format of `// CertExpireTime <does what>`. This also applies to the exported functions below.

Reply:
ack'd.

Reply:
added